recipe/meta.yaml

```diff
@@ -1,4 +1,6 @@
-{% set version = "5.7.0" %}
+{% set version = "5.7.0.dev20210429" %}  # PEP 386
```

Comment:
I only wonder how conda version comparison works... as ideally we'd like to have `5.7.0 > 5.7.0.dev`

[conda docs](https://conda.io/projects/conda/en/latest/user-guide/concepts/pkg-specs.html) say that `conda` uses [PEP 440](https://www.python.org/dev/peps/pep-0440/), which apparently honors this `.devN` scheme, so we're good here.

Reply:
Interesting, I found PEP 386 in https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#package-version .

Reply:
PEP 440 says "Developmental releases are ordered by their numerical component, immediately before the corresponding release (and before any pre-releases with the same release segment), and following any previous release (including any post-releases)."